src/templates/haml/get_started/index.html.haml

```diff
@@ -22,13 +22,13 @@
           = l('How to trade binaries?')
       .section-info
         = l('Follow a simple step-by-step guide to binary trading.')
-    .section
-      %a.pjaxload{:href => url_for('/get-started/beginners-faq')}
-        %div{:id => 'beginners-faq'}
+    .section.ja-hide
```

Comment:
Because hiding some of them made there be random white space in the middle
of the page so we moved all the hide to the bottom parts so there is no
white space
On 27 Aug 2015 13:43, "Raunak Kathuria" <notifications@github.com> wrote:

> In src/templates/haml/get_started/index.html.haml
> <https://github.com/binary-com/binary-static/pull/405#discussion_r38063882>
> :
>
> > @@ -22,13 +22,13 @@
> >            = l('How to trade binaries?')
> >        .section-info
> >          = l('Follow a simple step-by-step guide to binary trading.')
> > -    .section
> > -      %a.pjaxload{:href => url_for('/get-started/beginners-faq')}
> > -        %div{:id => 'beginners-faq'}
> > +    .section.ja-hide
>
> just want to know why we changed the order i.e change Random with
> Begineers Faq and vice versa
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/binary-com/binary-static/pull/405/files#r38063882>.
>